build.psm1

```diff
@@ -565,10 +565,13 @@ Fix steps:
             throw "RCEdit is required to modify pwsh.exe resources, please run 'Start-PSBootStrap' to install"
         }
 
-        $ProductVersion = (git --git-dir="$PSScriptRoot/.git" describe) -Replace '^v'
+        if (!$ReleaseTag)
+        {
+            $ReleaseTag = Get-PSCommitId
```

Comment:
you still need to remove the v at the beginning.
Get-PSVersion will do that for you, but I don't have a function to do this for the releasetag.